src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c

```diff
@@ -106,37 +106,37 @@ typedef enum {
 
 static DamageTable sDamageTable = {
     /* Deku Nut       */ DMG_ENTRY(0, POSISTERS_DAMAGEEFFECT_DEKUNUT),
-    /* Deku Stick     */ DMG_ENTRY(1, POSISTERS_DAMAGEEFFECT_NOEFFECT),
-    /* Horse trample  */ DMG_ENTRY(1, POSISTERS_DAMAGEEFFECT_NOEFFECT),
-    /* Explosives     */ DMG_ENTRY(1, POSISTERS_DAMAGEEFFECT_NOEFFECT),
-    /* Zora boomerang */ DMG_ENTRY(1, POSISTERS_DAMAGEEFFECT_NOEFFECT),
-    /* Normal arrow   */ DMG_ENTRY(1, POSISTERS_DAMAGEEFFECT_NOEFFECT),
-    /* UNK_DMG_0x06   */ DMG_ENTRY(0, POSISTERS_DAMAGEEFFECT_NOEFFECT),
-    /* Hookshot       */ DMG_ENTRY(1, POSISTERS_DAMAGEEFFECT_NOEFFECT),
-    /* Goron punch    */ DMG_ENTRY(1, POSISTERS_DAMAGEEFFECT_NOEFFECT),
-    /* Sword          */ DMG_ENTRY(1, POSISTERS_DAMAGEEFFECT_NOEFFECT),
-    /* Goron pound    */ DMG_ENTRY(0, POSISTERS_DAMAGEEFFECT_NOEFFECT),
-    /* Fire arrow     */ DMG_ENTRY(1, POSISTERS_DAMAGEEFFECT_NOEFFECT),
-    /* Ice arrow      */ DMG_ENTRY(1, POSISTERS_DAMAGEEFFECT_NOEFFECT),
+    /* Deku Stick     */ DMG_ENTRY(1, POSISTERS_DAMAGEEFFECT_NONE),
```

Comment:
```suggestion
    /* Deku Stick     */ DMG_ENTRY(1, POSISTERS_DMGEFF_NONE),
```
Everywhere in the repo uses `DMGEFF` for the abbreviation, and it looks like po_sisters are the one exception in master. This abbreviation would also apply to the new enum introduced in po_fusen